didcomm/errors.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+
+
+# TODO: finalize exceptions during development
+
+class DIDCommError(Exception):
+    pass
+
+
+class DIDNotResolvedError(DIDCommError):
```

Comment:
I more like `DIDNotFound` name.

Reply:
Actually I prefer DIDNotResolved. DID not found looks like we were searching for DID. But DID is there, it's given. We are not searching for a DID, we are resolving a given DID and searching for a DID DOC.
So, better names are either `DIDNotResolvedError` or `DIDDocNotFoundError`.